lnrpc/routerrpc/router_server.go

```diff
@@ -53,6 +56,10 @@ var (
 			Entity: "offchain",
 			Action: "write",
 		}},
+		"/routerrpc.Router/TrackPayment": {{
```

Comment:
Oh, yeah this is another benefit of splitting up the APIs that we can offer a way to only look at outcomes without initiating new payments :)

Reply:
Yes, we sometimes don't want to start if not started yet.

Reply:
I meant we can hand out a macaroon that only allows tracking, but not initiating payments :)

Reply:
Ah yes, didn't think of that, but indeed that is a benefit.